gpustack/schemas/stmt.py

```diff
@@ -92,3 +92,32 @@
 WHERE
     json_array_length(w.status::json->'gpu_devices') > 0
 """
+
+model_user_after_drop_view_stmt = "DROP VIEW IF EXISTS non_admin_user_models"
```

Comment:
Using a DB view for access control feels a bit overkill. Would it be more straightforward to handle this in the API layer with the existing tables?